pkgs/development/python-modules/tesserocr/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  # build dependencies
+  cython,
+  leptonica,
+  pkg-config,
+  tesseract,
+  # extra python packages
+  pillow
+}:
+
+buildPythonPackage rec {
+  pname = "tesserocr";
+  version = "2.5.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0nj33mwvppacy6p5mqk9a4x26hx9ailshgad84ks60wyms6rgjiv";
+  };
+
+  buildInputs = [ leptonica tesseract ];
+  nativeBuildInputs = [ cython pkg-config ];
```

Comment:
do you mind switching the order of build and nativeBuild inputs. 

this is a pretty typical ordering to a file:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/pandas/default.nix